components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -35,6 +38,8 @@ function FeedbackComponent(props: {
                 feedback: text,
                 href: window.location.href,
                 path: window.location.pathname,
+                error_object: props.error_object || undefined,
+                error_message: props.error_message,
```

Comment:
Not introduced in this PR, but this object can be explicitly typed as `EventProperties` .